app/src/main/java/com/example/mbanking/util/Variables.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.mbanking.util
+
+import com.example.mbanking.data.AccountData
+
+var defaultStartDate = "01.01.1970"
+var defaultEndDate = "30.01.3000"
+var listOfAccounts = mutableListOf(AccountData.First, AccountData.Second, AccountData.Third)
```

Comment:
replace var to val (or const val for start and end dates) because this values won't be changed in code.